### ISSUE TITLE
sprite for Royal Purple dragon mount is misplaced

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#8253]
As reported by Aerilita (3b724a41-48d3-4048-ae27-70b258cf5a72) in the Report a Bug guild:

"I just raised Royal Purple Dragon mount and I think the sprite for it is misplaced. I made a comparison with a regular Base Dragon mount here: http://savepic.net/8646829.gif "

**Comment by khipkin** [comment]
@Alys Would it be helpful to have an issue tag dedicated to image/pixel-art issues?

**Comment by Alys** [comment]
@khipkin Maybe... We're trying to limit the topic-specific tags as much as possible (we used to have HEAPS and it was hard to find the ones we wanted). We make an exception for translations because they're important for understanding the site, and also for challenges at the moment while there's so many challenge bugs, but for everything else we usually prefer to rely on keyword searching. In an ideal world, artwork bugs wouldn't stay unfixed for long enough for a tag to be very useful. :-/

**Comment by Hus274** [comment]
It looks like this is true for the peppermint dragon as well.

**Comment by Hus274** [comment]
And the spooky dragon.

**Comment by Hus274** [comment]
I think I can pick this up. I haven't done much for image manipulation but it doesn't look too bad.

**Comment by Alys** [comment]
@Hus274 Thank you! And for noticing the others.

**Comment by khipkin** [comment]
@Hus274 Those other sprites were already reported in #7982.  Maybe this issue should be merged into that one?

**Comment by Hus274** [comment]
@khipkin From what I saw on that issue, the list of images that needs to be fixed still needs to be compiled. In my preliminary look through the repo, I saw a fair number that still need to be fixed. My suggestion would be to make the list and then break down the work so anyone can pick up any fix. For this reason, I've fixed the ones mentioned above so they don't need to be compiled in that list and then picked up.
